strongbox-data-service/src/main/java/org/carlspring/strongbox/data/server/EmbeddedOrientDbServer.java

```diff
@@ -81,23 +198,26 @@ private void init()
 
         // prepare network configuration
         OServerNetworkConfiguration networkConfiguration = new OServerNetworkConfiguration();
+
         networkConfiguration.protocols = new LinkedList<>();
         networkConfiguration.protocols.add(binaryProtocol);
+
         networkConfiguration.listeners = new LinkedList<>();
         networkConfiguration.listeners.add(binaryListener);
 
         // add users (incl system-level root user)
         List<OServerUserConfiguration> users = new LinkedList<>();
         users.add(buildUser(connectionConfig.getUsername(), connectionConfig.getPassword(), "*"));
-        
+
         System.setProperty("ORIENTDB_ROOT_PASSWORD", connectionConfig.getUsername());
 
         // add other properties
         List<OServerEntryConfiguration> properties = new LinkedList<>();
         properties.add(buildProperty("server.database.path", getDatabasePath()));
         properties.add(buildProperty("plugin.dynamic", "false"));
-        properties.add(buildProperty("log.console.level", "info"));
-        properties.add(buildProperty("log.file.level", "fine"));
+        properties.add(buildProperty("log.console.level", "linfo"));
+        //properties.add(buildProperty("log.file.level", "fine"));
```

Comment:
If this is not required, could you please remove it?

Reply:
fixed